xml/storage_multipath.xml

```diff
@@ -364,7 +364,8 @@
     that the <filename>initrd</filename> and the installed system behave the
     same regarding the root file system and all other file systems required to
     boot the system. If Multipath is enabled in the system, it also needs to be
-    enabled in the <filename>initrd</filename> and vice versa. See <xref
+    enabled in the <filename>initrd</filename> and vice versa. See
+    <xref
```

Comment:
I personally find tags with their attributes on the next line really ugly. Especially like here, where the tag itself is now on an own line.